examples/clone.py

```diff
@@ -1,18 +1,37 @@
-#!/usr/bin/python
-# This trivial script demonstrates how to clone a remote repository.
-#
-# Example usage:
-#  python examples/clone.py git://github.com/jelmer/dulwich dulwich-clone
+"""Clone.
+
+This trivial script demonstrates how to clone a remote repository.
```

Comment:
Or lock

Reply:
Ok, Although I did not write this passage, I just keep it as it is.